.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+	"presets": [
+		"@babel/preset-env",
+		"@babel/preset-react"
```

Comment:
Add these plugins below:

@babel/plugin-proposal-object-rest-spread
@babel/plugin-proposal-class-properties
@babel/plugin-syntax-dynamic-import (Currently, @babel/preset-env is unaware that using import() with Webpack relies on Promise internally.)